api/src/main/java/io/minio/MakeBucketArgs.java

```diff
@@ -44,18 +44,18 @@ public static Builder newBuilder() {
   }
 
   public static final class Builder {
-    private BucketTemplate bucket;
+    private Bucket bucket;
     private String region;
```

Comment:
I think we need to keep region inside `Bucket` object. `Builder` would create appropriate `Bucket` object.